### PR TITLE
Fix lost audio when language changes

### LINKED ISSUE #8582: Loosing speaker audio after switching language
**Describe the bug**
When switching from one language to another while in a conf call, we loose the speaker and no longer can hear the other party. But mic still works and they can hear us! To fix one has to leave the call and reconnect again!

**To Reproduce**
1. Go to '.profile..'
2. Click on '...language .'
3. Scroll down to '....choose another language and hit save'
No longer can hear anyone’s voice

**Expected behavior**

**Actual behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [Chrome

### PR DESCRIPTION
**Pull request by Tainan404**
Closes #8582 and #8410.

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/audio/service.js [hunk 1]
```diff
@@ -69,7 +69,9 @@ export default {
   joinEchoTest: () => AudioManager.joinEchoTest(),
   toggleMuteMicrophone,
   changeInputDevice: inputDeviceId => AudioManager.changeInputDevice(inputDeviceId),
-  changeOutputDevice: outputDeviceId => AudioManager.changeOutputDevice(outputDeviceId),
+  changeOutputDevice: (outputDeviceId) => {
+    if (!AudioManager.initialized)AudioManager.changeOutputDevice(outputDeviceId);
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/audio/service.js]: Why do we check for this condition? Why only for Output device and not input for example?

Comment (Tainan404) [reply bigbluebutton-html5/imports/ui/components/audio/service.js]: Our output device is an audio tag on the client, but our input device are devices that are chosen by the user

Comment (Tainan404) [reply bigbluebutton-html5/imports/ui/components/audio/service.js]: https://github.com/bigbluebutton/bigbluebutton/blob/master/bigbluebutton-html5/imports/ui/components/audio/container.jsx#L141

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/audio/service.js]: Still not sure about this. Checking with @capilkey

**Comment by capilkey** [comment]
I just tested the PR and it doesn't actually work. It stops the audio from breaking on a locale change, but it also stops the ability to ever change your output device. Try plugging in speakers and a headset ,or two headsets. Then get to the echo test and say "No" and try switching the output device in Chrome and play the test audio.

On master it will play the test audio through whichever output source is selected. With this pull request it only plays the test audio out of the default output.

**Pull request closed without merging**